mod/research/set/self/research_page.rb

```diff
@@ -31,110 +31,4 @@ def slot_machine opts={}
       haml :slot_machine
     end
   end
```

Comment:
no changes here. I only separated code for the left and the right side of the research page